examples/atlas-dataLake-roles/aws-roles.tf

```diff
@@ -0,0 +1,64 @@
+resource "mongodbatlas_project" "test" {
+  name   = var.project_name
+  org_id = var.org_id
+}
+
+resource "mongodbatlas_cloud_provider_access" "test" {
+  project_id           = mongodbatlas_project.test.id
+  provider_name        = "AWS"
+  iam_assumed_role_arn = var.aws_iam_role_arn
```

Comment:
just curious about how this behaves. of course it needs the provider_access role, could you please elaborate a bit more?. i.e create a readme file in the  /atlas-dataLake-roles/README.md , describing for example all the steps , terraform apply , terraform refresh and so on :) thanks